packages/pangea-sdk/README.md

```diff
@@ -113,6 +113,8 @@ Verify that an event or a list of events has not been tampered with. Usage:
 
 ```
 usage: python -m pangea.verify_audit [-h] [--file PATH]
+or
+usage: poetry run python -m pangea.verify_audit [-h] [--file PATH]
```

Comment:
not sure if we should add this to the README. poetry shouldn't be mandatory and we're not referencing it anywhere else